### DISCUSSION TITLE
Submodules not found when fetching

### DISCUSSION THREAD

**Discussion by Rebelzob** [community/community#58546] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello everyone, 

I hope you can help me with this error its been driving me crazy these days, I got this error when trying to deploy my project with Github pages, the first deploy ran smoothly but after the first PR merge it never passed the checkout test again and its not letting me deploy, 

I have tried to add .gitmodules within my project adding the path and the URL, also tried to initialize the modules, update them, and even delete and start all over again, surely I am doing something wrong but it would be of a great help because not even redeploying the project its working, thanks a mil for your help!

Fetching submodules
  /usr/bin/git submodule sync --recursive
  /usr/bin/git -c protocol.version=2 submodule update --init --force --depth=1 --recursive
  Error: fatal: No url found for submodule path 'Portfolio' in .gitmodules
  Error: The process '/usr/bin/git' failed with exit code 128

**Comment by airtower-luna** [comment]
Can you link the repository and workflow log? Without seeing the configuration & log it's hard to say what may be wrong.

**Reply by Rebelzob** [reply]
Hello, @airtower-luna 

this is the link to the [repo](https://github.com/Rebelzob/Portfolio) and here is the link to the [logs](https://github.com/Rebelzob/Portfolio/actions/runs/5321707660)

thank you!

**Reply by airtower-luna** [reply]
That submodule looks highly suspicious, where is it *meant* to point? Having a `Portfolio` submodule in a `Portfolio` repository seems more like a mistake than anything that makes sense. Or is it supposed to be just a directory? :sweat_smile:

**Reply by Rebelzob** [reply]
it is suspicious, it's not meant to be there actually, let me delete it and see if that works 😄

**Comment by Rebelzob** [comment]
@airtower-luna, It worked!!! thanks a mil! ❤️
